### PR TITLE
Chart-Area-Visual feature | Topology

### PR DESCRIPTION
**Pull request by hemantsaini-7**
**Description:**

- Automation for Chart-Area-Visual feature | Topology

**Jira Id:**
     - [ODC-6659](https://issues.redhat.com/browse/ODC-6659) : topology-chart-area-visual
    

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
    export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.11-042801.devcluster.openshift.com/
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p topology
```
**Screen shots**:

chart-area-visual.feature
image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/topology/integration-tests/support/step-definitions/topology/chart-view.ts [hunk 1]
```diff
@@ -232,3 +237,253 @@ Then('user will see alert {string}', (alertName: string) => {
...
+      break;
+
+    case 'Channel':
+      cy.get(chartAreaPO.channelName).should('have.value', 'channel');
+      cy.get(chartAreaPO.submitButton).click();
+      break;
+
+    default:
+      break;
+  }
```
Comment (sanketpathak) [inline frontend/packages/topology/integration-tests/support/step-definitions/topology/chart-view.ts:435]: Make a function for it and then use it here

File: frontend/packages/topology/integration-tests/support/step-definitions/topology/chart-view.ts [hunk 2]
```diff
@@ -232,3 +237,253 @@ Then('user will see alert {string}', (alertName: string) => {
...
+      break;
+    case 'Event Source':
+      cy.get(chartAreaPO.eventsource).click();
+      break;
+    case 'Channel':
+      cy.get(chartAreaPO.channel).click();
+      break;
+    default:
+      break;
+  }
```
Comment (sanketpathak) [inline frontend/packages/topology/integration-tests/support/step-definitions/topology/chart-view.ts:352]: Make a function for it

**Comment by sanketpathak** [comment]
Test runs are green 
/lgtm

**Comment by hemantsaini-7** [comment]
This PR only affects our e2e tests.
/label px-approved
/label docs-approved
/label qe-approved

**Pull request merged**